lib/init.js

```diff
@@ -65,27 +69,28 @@ var installPackage = function (pkgName) {
   })
 }
 
-var validatePattern = function (pattern) {
+function validatePattern (pattern) {
   if (!glob.sync(pattern).length) {
     log.warn('There is no file matching this pattern.\n')
   }
 }
 
-var validateBrowser = function (name) {
+function validateBrowser (name) {
   // TODO(vojta): check if the path resolves to a binary
   installPackage('karma-' + name.toLowerCase().replace('canary', '') + '-launcher')
 }
 
-var validateFramework = function (name) {
+function validateFramework (name) {
   installPackage('karma-' + name)
 }
 
-var validateRequireJs = function (useRequire) {
+function validateRequireJs (useRequire) {
   if (useRequire) {
     validateFramework('requirejs')
   }
 }
 
+// TODO: update tests to not rely on internal module state and change this to const
```

Comment:
The mocking library used in tests does not expose block-scoped declarations, so I had to keep `var` here. What do you think about exposing some of the methods from this file on `module.exports` to get rid special import in tests completely?

Reply:
Yes, that is better....again a separate PR ;-)